File: C++/0013. Table.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.

int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
    int n,m;
    while(cin>>n>>m){
        string s1 = "";
        string s2 = "";
        for(int j=0;j<=4*m;++j){
            if(j%4){
                s1 += '-';
                s2 += ' ';
            }
            else{
                s1 += '+';
                s2 += '|';
            }
        }
        for(int i=0;i<=2*n;++i){
            cout<<(i%2?s2:s1)<<endl;
        }
    }
    return 0;
}

File: Python/0013. Table.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:
            n,m = map(int,input().strip().split())
            s1 = ""
            s2 = ""
            for j in range(4*m+1):
                if j%4:
                    s1 += '-'
                    s2 += ' '
                else:
                    s1 += '+'
                    s2 += '|'
            for i in range(2*n+1):
                print(s2 if i%2 else s1)
            return
            
        except EOFError:
            break
    
if __name__ == "__main__":
    func()
